Pasing from SAS to R second program

```diff
@@ -0,0 +1,102 @@
+Moving Second Program from SAS to R
+
+This are the notes that apper as I move forward. I have the code in a separate document. 
+
+
+line 28 of SAS, I don't know why are we deleting the data that is NOT 1. 
+
```

Comment:
inv_seq ("inventor sequence") denotes in what order the inventors are listed on the patent. In the inventors dataset, there is one observation for each inventor listed on each patent, so many patents have multiple entries (one for each inventor on the patent). This step discards all the entries for the second, third, fourth-listed (etc.) inventors so we have just one entry for each patent, and it is the one for the first-listed inventor.